packages/cadl-ranch-specs/http/authentication/http/custom/mockapi.ts

```diff
@@ -8,7 +8,7 @@ const validAndInvalidScenarios = getValidAndInvalidScenarios(
   "http/custom",
   "invalid-api-key",
   function addOptionalParamOldApiVersionNewClientValidate(req: MockRequest): void {
-    req.expect.containsHeader("Authorization", "SharedAccessKey valid-key");
+    req.expect.containsHeader("authorization", "SharedAccessKey valid-key");
```

Comment:
Is this method always take lower-case?

Reply:
Yes. Refer: https://microsoft.github.io/PowerBI-JavaScript/interfaces/_node_modules__types_node_http_d_._http_.incominghttpheaders.html.